client/src/document/writer-toolbar/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Doc } from "../types";
+import { EditButtons } from "./edit-buttons";
+import { ToggleDocumentFlaws } from "./flaws";
+import DocumentSpy from "./spy";
+
+export default function WriterToolbar({
+  doc,
+  onMessage,
+}: {
+  doc: Doc;
+  onMessage: Function;
+}) {
+  return (
+    <div style={{ padding: "10px 0" }}>
```

Comment:
‘style’ is meant for hacks or exceptions. I’d prefer a className and a dedicated .scss file.

Reply:
Fair, I thought this is throwaway stuff that Schalk will waltz away anyway, but if your preference is to have none of it, I'll gladly kick it out